### ISSUE TITLE
Annotations in Gallery

### ISSUE THREAD

**Issue by haraldeb** [ruven/iipmooviewer#67]
We, the State Library of Vorarlberg (Austria) use the iipmooviewer-gallery for our digital collections. 
e.g.: http://pid.volare.vorarlberg.at/o:61874

Is it possible to show annotations in a gallery?

**Comment by ruven** [comment]
No, this is not currently implemented, but it should be easy to add this. I'll get back to you soon when I've had a chance to look into it.

**Comment by haraldeb** [comment]
Thank you for your fast response, Ruven. And thank you for that great Software :)

**Comment by haraldeb** [comment]
Hi Ruven, 
did you had the chance to look at this issue?

**Comment by abubelinha** [comment]
Hi @haraldeb @ruven , any advances so far? 
This sounds interesting to me too.
